app/models/energy/materials.py

```diff
@@ -813,3 +813,49 @@ class EnergyWindowMaterialShade (BaseModel):
         ' If air cannot pass through the shade material, airflow_permeability = 0.'
         ' Default value is 0.'
     )
+
+
+class WindowConstruction(BaseModel):
```

Comment:
@chriswmackey Should `WindowConstruction` be added in `materials.py`? I had added this to `construction.py`.